src/lib.rs

```diff
@@ -734,6 +732,9 @@ impl Picture {
 unsafe impl Send for Picture {}
 unsafe impl Sync for Picture {}
```

Comment:
@lu-zero  Do you want this to be removed and implicitly implemented? Same question for `Plane`.

Reply:
Sounds a good idea. Thank you :)